### DISCUSSION TITLE
Adding custom parameter to spec

### DISCUSSION THREAD

**Discussion by aeryz** [ChainSafe/lodestar#6683] [Q&A] [answered]
I couldn't find a way to define a custom parameter that is going to be returned with `/eth/v1/config/spec` API. Is this possible or it does not allow extending the defined keys?

**Comment by nflaig** [comment, marked as answer]
Unknown values are either filtered out if provided via `--paramsFile`, or would throw an error on startup if passed via `--params.*`.

What's your use case to return unused / unspec'd params from `/eth/v1/config/spec` API?

**Reply by aeryz** [reply]
I modified one of the beacon api's and wanted to use an additional spec in order to distinguish between a regular lodestar vs the custom one but I guess I can just add a new field programmatically since I already have a fork. Thanks for help!
